### PR TITLE
Add Modal for vocabulary

### PR DESCRIPTION
**Pull request by tobihoff**

### CODE REVIEW

File: client/src/components/Modals/Modal.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import RemoveIcon from '../../icons/RemoveIcon';
+import Badge from '../Badges/Badge';
+
+const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: flex;
```
Comment (holgermanzke) [inline client/src/components/Modals/Modal.js]: row??

File: client/src/components/Modals/Modal.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  flex-wrap: wrap;
+  position: absolute;
+  min-width: 175px;
```
Comment (holgermanzke) [inline client/src/components/Modals/Modal.js]: min-width: 175px and max-width: 175px => width: 175px?

File: client/src/components/Modals/Modal.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  min-height: 450px;
```
Comment (holgermanzke) [inline client/src/components/Modals/Modal.js]: same here

Comment (tobihoff) [reply client/src/components/Modals/Modal.js]: forget to change, thanks

**Pull request merged**